docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -60,6 +60,52 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3.
 {{% /admonition %}}
 
+## Add dashboard and folder permissions to service accounts
```

Comment:
I think you mean for these to be under the **Authentication and authorization** heading so I'm making all of these H3s.
```suggestion
### Add dashboard and folder permissions to service accounts
```